doc/adoc/user_guide.adoc

```diff
@@ -144,6 +144,17 @@ If `use_zypper_migration: false` is set, the value for the
 `migration_product` will not be effective because it's used only in
 combination with the Zypper migration workflow.
 
+Preserve System Data::
+Preserve custom data file(s) e.g udev rules from the system
+to be migrated into the upgrade live system and make sure
+they will become effective.
```

Comment:
I think we should point out a general limitation with this. Such as

"""
udev rules that require custom drivers will not have the desired effect as the migration system will not include these drivers and therefore execution of those rules will fail. Rules with such properties should not be listed.
"""

Reply:
yes very good point, thanks